Dayanand Viveki/HTML/Assignment5/biodata.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="assets/css/style.css" />
+        <title>Learn HTML,CSS extrenal stylesheet</title>
```

Comment:
always add space after comma make this change everywhere

Reply:
ok mdm